easybuild/scripts/bootstrap_eb.py

```diff
@@ -66,7 +66,8 @@
 PYPI_SOURCE_URL = 'https://pypi.python.org/packages/source'
 
 VSC_BASE = 'vsc-base'
-EASYBUILD_PACKAGES = [VSC_BASE, 'easybuild-framework', 'easybuild-easyblocks', 'easybuild-easyconfigs']
+VSC_INSTALL = 'vsc-install'
+EASYBUILD_PACKAGES = [VSC_BASE, VSC_INSTALL, 'easybuild-framework', 'easybuild-easyblocks', 'easybuild-easyconfigs']
```

Comment:
ah, yes, crap, I'm making the same mistake OpenHPC did, cfr. https://github.com/openhpc/ohpc/commit/3f9442a6b53938e1d31cda70dae7d7a66e9c2827 ;)